"0","data_2022 = read.csv(file = 'pbp-2022')
"
"2","Warning: cannot open file 'pbp-2022': No such file or directory"
"2","Error in file(file, ""rt"") : cannot open the connection
"
